.circleci/config.yml

```diff
@@ -135,24 +138,33 @@ commands:
           command: |
             aws configure --profile sdk_s3_release set region us-east-1
             echo -e "[sdk_s3_release]\naws_access_key_id=${S3_RELEASE_AWS_ACCESS_KEY_ID}\naws_secret_access_key=${S3_RELEASE_AWS_SECRET_ACCESS_KEY}\n" >> ~/.aws/credentials
+  pre_start_simulator:
+    description: >-
+      pre start simulator, build may fail is simulator is not started
```

Comment:
is it possible to wait until the simulator comes up?

Reply:
the command used here will wait  until the simulator comes up